bridge/tfchain_bridge/pkg/bridge/mint.go

```diff
@@ -23,18 +26,30 @@ func (bridge *Bridge) mint(ctx context.Context, senders map[string]*big.Int, tx
 	}
 
 	if minted {
-		log.Info().Str("tx_id", tx.Hash).Msg("transaction is already minted")
+		logger.Info().
```

Comment:
I am not sure if we should log before return every error, or just log when we receive an error. If the caller to `mint` function now decide to log the returned error we will have double logs.

What I do is that I always log an error if i can't bubble it up more, the error then can have all the information we need to build a detailed log 

Otherwise we can endup with the same error printed multiple times